blocks/variables.ts

```diff
@@ -60,18 +60,19 @@ export const blocks = createBlockDefinitionsFromJsonArray([
     ],
     'previousStatement': null,
     'nextStatement': null,
+    'output': null,
```

Comment:
Output isn't needed here; this is the setter, which is just a statement block without any output value.